modules/lx-instance/watchmaker-lx-instance.template.cfn.yaml

```diff
@@ -435,8 +441,11 @@ Resources:
             02-extract-cloudwatch-agent:
               command: >-
                 yum -y install unzip &&
-                unzip /etc/cfn/scripts/AmazonCloudWatchAgent.zip -d
-                /etc/cfn/scripts/aws-cw-agent
+                unzip /etc/cfn/scripts/AmazonCloudWatchAgent.zip -d /etc/cfn/scripts/aws-cw-agent
```

Comment:
If changing this line here, let's be consistent and change it in all the other templates as well.

Reply:
Updated. Suppose I should make changes like these consistent across the board.. Do we have a set line length we'd like to stick to across these templates?

Reply:
More just a matter of consistency and readability than an adherence to a particular line length. In pure python, the restriction on line length should be seen as a warning that the code could use refactoring to simplify the implementation. It's likely any line length violation is a result of too many nested blocks, which are a code smell.

In this case, in yaml, there's not a lot of refactoring we can do to limit line length that also improves readability. We could perhaps stick to a hard 80 chars, which can be easier to scroll through in most terminals, but such changes should be handled in a separate PR, and should be enforced through a CI test.